### ISSUE TITLE
rayCast property not working

### ISSUE THREAD

**Issue by UnidayStudio** [UPBGE/blender#671]
The propertie argument of the rayCast isn't working on 0.2.3 anymore... If you try to rayCast to a certain prop, it will return an object even if there is no props on it...

```
from bge import logic

def a(cont):
    obHit,_,_ = cont.owner.rayCast([0,0,0],[0,10,0],0.0,"test",1,1,0)
    
    if obHit != None:
        print(obHit)
```

If you run this code and put an object without the "test" prop, it will print them. :/

**Comment by panzergame** [comment]
I could not confirm the bug.

My example file is : http://pasteall.org/blend/index.php?id=49589.

**Comment by UnidayStudio** [comment]
Take a look right now: It will print "Suzanne". but there is no "test" prop on the Suzanne...
http://pasteall.org/blend/index.php?id=49590

**Comment by lopas85** [comment]
someone know if has been released a 0.2.3 upbge's version where rayCast function is ok? I need to continue some projects that are incompatible with the 0.2.4 and also with the 0.2.3+render attachments
